shared/js/project.js

```diff
@@ -127,7 +111,7 @@ module.exports = {
       $('.project_updated_at').text("Not saved"); // TODO i18n
     }
   },
-  appToProjectUrl: function () {
+  appToProjectUrl_: function () {
```

Comment:
Can these be made truly private instead of exporting them with a trailing underscore?  For `appToProjectUrl_` at least it looks like it doesn't need access to the `this` context.